docs/src/main/sphinx/language/types.rst

```diff
@@ -227,8 +229,11 @@ The following examples illustrate the behavior::
 Instant in time that includes the date and time of day with ``P`` digits of
 precision for the fraction of seconds and with a time zone. Values of this
 type are rendered using the time zone from the value.
+Time zones are expressed as the numeric UTC offset value::
```

Comment:
This turned out to be understood as indicating that Trino supports numeric offset zones, but not named zones.
see https://github.com/trinodb/trino-python-client/pull/229#discussion_r954018180 cc @jhlodin 

we need to improve the wording to clarify named zones are supported

cc @mosabua

Reply:
Probably we need to mention this separately both TIME and TIMESTAMPTZ since TIME doesn't support named zones (since they don't make sense there) while TIMESTAMPTZ does.

Reply:
Whatever supports timezone should be consistent .. is that the case now.. I am not sure to be honest.

Reply:
It cannot be consistent since TIME has no concept of named zones. What does `TIME '01:01:01' America/Los_Angeles` mean? Depending on what day it was `America/Los_Angeles` can mean different things.

We need to explicitly make it more clear that `TIME` with timezone supports only offsets and not named zones while `TIMESTAMP` with timezone supports both named offsets and numbered offsets.

Reply:
> What does `TIME '01:01:01' America/Los_Angeles` mean?

This is a wrong question to ask. We don't need to go into details why.

> TIME has no concept of named zones. 

yes, it does not.

Reply:
`time(p) with time zone` is in the SQL standard. It's defined as a time with a time zone displacement containing an hour and a minute component in the range -14:00 to +14:00. That's the *only* concept of timezones supported by the specification, even for the `timestamp(p) with time zone type`. Support for political time zones is an extension to the standard.

Reply:
So .. to simplify for users and consistency .. should we remove the support for political time zones everywhere?

Reply:
No, political timezones are useful and necessary for `timestamp(p) with time zone`.

Reply:
Okay ... I think we have enough info now to run with a doc update PR .. @jhlodin will submit soon.

Reply:
Proposed type docs change here: https://github.com/trinodb/trino/pull/13927